htdocs/modules/system/themes/modern/modern.php

```
@@ -390,13 +390,16 @@ private function buildMenu(&$tpl)
         }
 
         // Detect whether system panel should be opened
-        $requestUri = \Xmf\Request::getString('REQUEST_URI', '', 'SERVER');
-        $path       = (string) parse_url($requestUri, PHP_URL_PATH);          // strip query string
+        $requestUri  = \Xmf\Request::getString('REQUEST_URI', '', 'SERVER');
+        $pathUri     = (string) parse_url($requestUri, PHP_URL_PATH);          // strip query string
+        $httpReferer = \Xmf\Request::getString('HTTP_REFERER', '', 'SERVER');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Xmf\Request' in method 'buildMenu'. (undefined)

(StaticAccess)